### ISSUE TITLE
Change Star Data menu

### ISSUE THREAD

**Issue by ljramones** [ljramones/trips#183]
The “star data” menu.  Currently looks like this

<img width="290" alt="image" src="https://user-images.githubusercontent.com/226595/106224910-1e93ae80-61b2-11eb-954f-e963de2a8ebe.png">

This is WAY better than it was, but I think it needs more changes.  I propose the following:  
•	Select Active Dataset (grayed out if no datasets have been imported yet.) 
•	Import dataset from a file
•	View/Edit selected Star data 
•	Import Multiple       (do we REALLY want this? I question it. ) 
•	Export current selection to Excel 
•	Create a new data set from existing. 

In other words, I want to seperate “Importing” from “Choosing” at the menu level 

I can do this.
Having Select active data set means you want to put the functionality out of the current dialog and create an new one for this functionality 

I can remove this. At one point I was iterating through all the possibilities

**Comment by cegannon1** [comment]
My impression that

a) this is too much plumbing in one list
b) some less frequently employed functions (projected) should be suboridated under a category ~ "other functuons"
c) some of the teminology needs to be more "average user" friendly.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by cegannon1** [comment]
I tried to post to Github but am on phone and username and password is still not syncing.

My comment, again:

"My impression that

a) this is too much plumbing in one list
b) some less frequently employed functions (projected) should be suboridated under a category ~ "other functuons"
c) some of the teminology needs to be more "average user" friendly."

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ljramones** [comment]
ok, I can make a first cut and then we can re-evaluate and choose next steps

**Comment by ljramones** [comment]
ok, I have a first cut and will mark it as done until we take a look at what I have completed
